packages/metronome/buildinfo.json

```diff
@@ -2,8 +2,8 @@
   "requires": ["java", "exhibitor"],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://downloads.mesosphere.com/maven/dcos/metronome/0.2.3/metronome-0.2.3.tgz",
-    "sha1": "ef42cb936a59360ecd0dda3a434f75e1007f3527"
+    "url": "http://downloads.mesosphere.io/metronome/snapshots/metronome-0.2.4.tgz",
```

Comment:
Whats wrong with https?

Reply:
I wasn't aware we have a preference... I pasted what I was working with locally.   I can default to https.

Reply:
defaulting to `https` better.

Reply:
yeah.. this was a mistake... good catch @orsenthil the metronome bump was suppose to be on the other PR.   Let me update this again.  This should be just the new mom test.